tap_pardot/sync.py

```diff
@@ -16,3 +18,37 @@ def sync(client, config, state):
 
         for rec in stream_object.sync():
             singer.write_record(stream_id, rec)
+
+
+def sync_properties(client: Client):
```

Comment:
Also, really good call on making this just a separate function, rather than extending the existing sync!